Add documentation for issuer selection capability for organization applications (#6070)
wso2/docs-is: en/includes/guides/organization-management/organization-applications.md

{% if product_name == "Asgardeo" or (product_name == "WSO2 Identity Server" and is_version > "7.2.0" ) %}

!!! note

    By default, applications created within an organization use the root organization's token issuer. As an organization administrator, you can change this behavior and configure an application to use either the root organization's token issuer or your organization’s own token issuer.

>     For more information on the use cases to select the token issuer, see [Selecting the appropriate token issuer for organization applications]({{base_path}}/guides/organization-management/select-token-issuer-for-organization-apps).

Reviewer: ```suggestion
    For guidance on selecting a token issuer, see [Select token issuer for organization applications]({{base_path}}/guides/organization-management/select-token-issuer-for-organization-apps).
```
